### ISSUE TITLE
Trying to set this up.

### ISSUE THREAD

**Issue by gzion7** [awslabs/cloudwatch-logs-subscription-consumer#5]
I have cloudwatch logs and an elasticsearch cluster w/kibana. I'm trying to setup cloudwatch-logs-subscription-consumer found from this repo.

But unfortunately it doesn't really tell me HOW this is setting this up. I know there is a cloudformation template, however that template is not an option.
Do you have any other instructions that can help?

**Comment by danielcbright** [comment]
:+1: agreed, there needs to be more documentation on how to setup an existing ELK stack with this config.

**Comment by fraser-m** [comment]
Agreed to the above, the setup for elasticsearch means that using the Managed Service or the cloudformation templates provided is not an option.

It would be very very useful to be able to take the logs from Cloudwatch into our stack for visualisation.

**Comment by danielcbright** [comment]
I have managed to do some hacking on this to get it printing valid json data to stdout, which logstash can then pick up and run with, however it is not pretty or ideal..... I'd still like to see some guide on how to implement this without using AWS's specific implementation...

**Comment by ckelner** [comment]
Would be extremely helpful :dancer:

**Comment by danielcbright** [comment]
Added a new project here on how to make this work cross-account/region, just has a diagram right now but I'll update with example scripts later:

https://github.com/danielcbright/cross-account-cloudwatch-logs-to-kinesis

**Comment by amitkulkarni1712** [comment]
I set it up using the template & its up .. But i am not able to view the data on any of the dash ..is their any step by step document which can help me achieve the screens as in screenshots in article.

**Comment by NareshDealer** [comment]
@fraser-m  Have  you found any solution for sending cloudwatch logs to your own stack.?

**Comment by berry2012** [comment]
Is it possible to get a step by step guide on how to deploy this beautiful solution? I plan on using the template directly.
